### ISSUE TITLE
[BUG] Config file saving from UI causes "500 Internal Server Error"

### ISSUE THREAD

**Issue by 0FiRE0** [bunkerity/bunkerweb#643]
### What happened?

At a given point in time, some of the directories inside the `data` volume disappear out of a sudden.
Usually, the `/data/configs/` directory includes multiple sub-directories when the containers are created for the first time, but sometimes they disappear, after that, when creating any new config file from the Web UI inside any of the missing sub-directories, a 500 Error shows up and no config files are created anymore.

A snapshot of the current `configs` content:
<img width="495" alt="image" src="https://github.com/bunkerity/bunkerweb/assets/19993814/7d93d683-57f9-48db-90c0-072e8c0acda7">

Noting that from within the Web UI, it looks like all directories are shown (the directory structure might be stored in a DB or so, without querying the actual filesystem).
<img width="1124" alt="image" src="https://github.com/bunkerity/bunkerweb/assets/19993814/90fdba4c-ed44-425e-99d2-f0ba163bba46">

After creating any file inside one of the non-existing directories (E.g. modsec), a `500 Internal Server Error` shows up, and no file is created.
Terminal logs are not very helpful, as they only show 500 error without any details.

<img width="1467" alt="image" src="https://github.com/bunkerity/bunkerweb/assets/19993814/ce2d537b-40ee-46fb-ae79-a91015a5f841">

### How to reproduce?

Not sure how to reproduce it exactly, but this happened with me randomly after re-creating or restarting the containers.

### Configuration file(s) (yaml or .env)

```YAML
version: "3.5"

services:
  mybunker:
    image: bunkerity/bunkerweb:1.5.2
    ports:
      - 80:8080
      - 443:8443
    environment:
      - SERVER_NAME=example.com # replace with your domain
      - MULTISITE=yes
      - AUTO_LETS_ENCRYPT=no
      - DISABLE_DEFAULT_SERVER=yes
      - USE_CLIENT_CACHE=yes
      - USE_GZIP=yes
      - API_WHITELIST_IP=127.0.0.0/8 10.20.30.0/24
      - example.com_USE_UI=yes
      - example.com_SERVE_FILES=no
      - example.com_USE_REVERSE_PROXY=yes
      - example.com_REVERSE_PROXY_URL=/admin # replace with another url
      - example.com_REVERSE_PROXY_HOST=http://bw-ui:7000
      - example.com_INTERCEPTED_ERROR_CODES=400 404 405 413 429 500 501 502 503 504
    labels:
      - "bunkerweb.INSTANCE"
    networks:
      - bw-universe
      - bw-services

  bw-scheduler:
    image: bunkerity/bunkerweb-scheduler:1.5.2
    depends_on:
      - mybunker
    environment:
      - DOCKER_HOST=tcp://bw-docker-proxy:2375
    volumes:
      - bw-data:/data
    networks:
      - bw-universe
      - bw-docker

  bw-ui:
    image: bunkerity/bunkerweb-ui:1.5.2
    depends_on:
      - bw-docker-proxy
    environment:
      - DOCKER_HOST=tcp://bw-docker-proxy:2375
      - ADMIN_USERNAME=admin
      - ADMIN_PASSWORD=123456!Aa # replace with a stronger password
    volumes:
      - bw-data:/data
    networks:
      - bw-universe
      - bw-docker

  bw-docker-proxy:
    image: tecnativa/docker-socket-proxy:0.1
    volumes:
      - /var/run/docker.sock:/var/run/docker.sock:ro
    environment:
      - CONTAINERS=1
      - LOG_LEVEL=warning
    networks:
      - bw-docker

volumes:
  bw-data:

networks:
  bw-universe:
    name: bw-universe
    ipam:
      driver: default
      config:
        - subnet: 10.20.30.0/24
  bw-services:
  bw-docker:
```

### Relevant log output

```shell
bunkerweb-bw-ui-1            | 172.25.0.1 - - [17/Sep/2023:11:36:37 +0000] "POST /admin/configs HTTP/1.0" 500 255 "http://example.com/admin/configs" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:109.0) Gecko/20100101 Firefox/117.0"
bunkerweb-mybunker-1         | example.com 172.25.0.1 - - [17/Sep/2023:11:36:37 +0000] "POST /admin/configs HTTP/1.1" 500 703606 "http://example.com/admin/configs" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:109.0) Gecko/20100101 Firefox/117.0"
```

### BunkerWeb version

1.5.2

### What integration are you using?

Docker

### Removed private data

- [X] I have removed all private data from the configuration file and the logs

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by 0FiRE0** [comment]
As a side note, if the directory has already existed (E.g. inside modsec-csr), the user can still add files from the UI successfully.

I tried creating the missing directories manually from the terminal and granting them the same permissions and access levels. However, it is still returning 500 errors, and deleting the newly created directories by itself.

**Comment by TheophileDiot** [comment]
Hi @0FiRE0, thank you for opening this issue, I'll look into it and let you know

**Comment by fl0ppy-d1sk** [comment]
Hello @0FiRE0,

It should be fixed in the last 1.5.3 release. Don't hesitate to open a new issue if needed.
